packages/core/usage-data/core-usage-data-server-internal/index.ts

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+export { CoreUsageDataService, CoreUsageStatsClient } from './src';
+export type { InternalCoreUsageDataSetup } from '@kbn/core-usage-data-base-server-internal';
```

Comment:
Re-exporting the internal contract type from the base package for consistency, given all other domains allow to import the service impl and the associated contracts from the same package.